test/autests/gold_tests/proxy-protocol/replay_files/http_single_transaction_no_pp.replay.yaml

```diff
@@ -0,0 +1,85 @@
+meta:
```

Comment:
This should have a coypright as well...though I think Fei pointed out that some don't. For new replay yaml files at least we should add them.

Reply:
Added.